lib/api/src/sys/module.rs

```diff
@@ -267,11 +267,11 @@ impl Module {
     /// # }
     /// ```
     pub unsafe fn deserialize(
-        engine: &impl AsEngineRef,
+        store: &impl AsStoreRef,
         bytes: impl IntoBytes,
     ) -> Result<Self, DeserializeError> {
         let bytes = bytes.into_bytes();
-        let artifact = engine.as_engine_ref().engine().deserialize(&bytes)?;
+        let artifact = store.as_store_ref().engine().deserialize(&bytes)?;
         Ok(Self::from_artifact(artifact))
```

Comment:
Why puting store back instead of engine?

Reply:
This is aligned with master again.